src/picknik_ur_site_config/objectives/open_door_fixed_handle_ml.xml

```diff
@@ -0,0 +1,71 @@
+      <SubTree ID="Re-Zero Force-Torque Sensors" _collapsed="true"/>
+      <SubTree ID="Update Admittance Controller" _collapsed="true"/>
+      <!-- Look at the desired location where objects (door) will be present -->
+      <Action ID="MoveToWaypoint" waypoint_name="Right Wall" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="true" constraints=""/>
+      <SubTree ID="GetDoorGraspableObjectSubtree" _collapsed="true"/>
+
+      <!-- Get the masks for the handle subframe of the door GraspableObject -->
+      <Control ID="Fallback">
+        <Control ID="Sequence">
+          <Action ID="MoveToWaypoint" waypoint_name="Right Wall" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="false" constraints=""/>
+          <SubTree ID="GetHandleGraspableObjectSubtree" _collapsed="true"/>
+        </Control>
+        <Control ID="Sequence">
+          <Action ID="MoveToWaypoint" waypoint_name="Extended Right" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="false" constraints=""/>
+          <SubTree ID="GetHandleGraspableObjectSubtree" _collapsed="true"/>
+        </Control>
+      </Control>
+
+      <!-- Get the masks for the hinge subframes of the door GraspableObject -->
+      <Control ID="Fallback">
+        <Control ID="Sequence">
+          <Action ID="MoveToWaypoint" waypoint_name="Right Wall" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="false" constraints=""/>
+          <SubTree ID="GetHingeGraspableObjectSubtree" _collapsed="true"/>
+        </Control>
+        <Control ID="Sequence">
+          <Action ID="MoveToWaypoint" waypoint_name="Hinge View" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="false" constraints=""/>
+          <SubTree ID="GetHingeGraspableObjectSubtree" _collapsed="true"/>
+        </Control>
+      </Control>
+
+      <!-- Helper behavior to convert poses into subframes that adhere to the expected axis convention -->
+      <Action ID="GetMoveAlongArcSubframes" target_grasp_pose="{grasp_pose}" hinge_axis_pose_start="{screw_1_pose}" hinge_axis_pose_end="{screw_2_pose}" move_along_arc_subframes="{move_along_arc_subframes}"/>
+      <!-- Add the subframes to the Object-->
+      <Decorator ID="ForEachObjectSubframe" vector_in="{move_along_arc_subframes}" out="{subframe}">
+        <Control ID="Sequence">
+          <Action ID="AddSubframeToObject" subframe="{subframe}" graspable_object="{graspable_object}"/>
+        </Control>
+      </Decorator>
+      <!-- Now for each door graspable object (with subframes added) do MTC -->
+      <Decorator ID="ForEachGraspableObject" vector_in="{graspable_objects}" out="{graspable_object}">
```

Comment:
As discussed earlier, maybe for now consider including the "Get First Graspable Object in List" subtree from https://github.com/PickNikRobotics/advanced_objective_training_config/blob/main/objectives/get_first_graspable_object_in_list.xml and forgetting the loop.